ServerCore/Pages/Events/Standings.cshtml.cs

```diff
@@ -112,6 +112,12 @@ public async Task OnGetAsync(SortOrder? sort)
                 case SortOrder.ScoreDescending:
                     teamsFinal = teamsFinal.OrderByDescending(ts => ts.Score).ThenByDescending(ts => ts.Rank).ThenByDescending(ts => ts.Team.Name).ToList();
                     break;
+                case SortOrder.HintsEarnedAscending:
+                    teamsFinal = teamsFinal.OrderBy(ts => ts.Team.HintCoinsEarned).ThenBy(ts => ts.Rank).ThenBy(ts => ts.Team.Name).ToList();
```

Comment:
OrderBy ascending HintCoinsEarned (weaker to stronger team order)
ThenBy ascending Rank (stronger to weaker team order)

I think this file has a few of these ThenBys that break "conventional logic"